assets/activation-scripts/activate.d/generate-fish-startup-commands.bash

```diff
@@ -53,6 +43,10 @@ generate_fish_startup_commands() {
   # Set the prompt if we're in an interactive shell.
   echo "if isatty 1; source '$_activate_d/set-prompt.fish'; end;"
 
+  # We already customized the PATH and MANPATH, but the user and system
+  # dotfiles may have changed them, so finish by doing this again.
+  echo "$_flox_env_helper fish | source;"
```

Comment:
I think this line can be `$_flox_env_helper fish` instead of `echo "$_flox_env_helper fish | source;"`